lib/Runtime/Base/ThreadContext.cpp

```diff
@@ -798,8 +798,13 @@ ThreadContext::FindPropertyRecord(Js::JavascriptString *pstName, Js::PropertyRec
         return;
     }
 
-    LPCWSTR psz = pstName->GetSz();
+    LPCWSTR psz = pstName->GetString();
```

Comment:
Maybe add a comment here explaining why this is safe? Also, perhaps rename the variable since its no longer a zero-terminated string.